classes/Tools.php

```diff
@@ -3419,7 +3419,7 @@ public static function dieOrLog($msg, $die = true)
      *
      * @return string New string
      */
-    public static function nl2br($str)
+    public static function nl2br(string $str)
```

Comment:
Please alert the BC break in your PR description :)

Reply:
I don't understand why you consider this a BC break, since Scalar type declaration was introduced in PHP 7.0<sup>[1](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.scalar-type-declarations)</sup>, and according to Prestashop's PHP compatibility chart, current active (1.7.8) and stale (1.7.7) branches need PHP 7.1 or later<sup>[2](https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/#php-compatibility-chart)</sup>.